py_openda/examples/swan_calibration_dud/py_openda/algorithms/Dud.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct 10 14:35:15 2018
```

Comment:
Op deze plek kan je wat typen over het algoritme en vergeet ook niet een verwijzing te maken naar het paper.
Verder dezelfde opmerking als bij je ander commit dat bij alle functies een stukje uitleg getyped moet worden.

O, heb je al je code door pylint gehaald. Deze statische analyse is erg zinvol.